README.md

```diff
@@ -0,0 +1,17 @@
+# vpphelper
+
+vpphelper provides helper functions (using govpp) that helps to connect to VPP and configure VPP using a configuration (.conf) file.
+
+`StartAndDialContext` function starts VPP using a VPP configuration file and returns a `Connection` to VPP.
+
+```go
+conn, vppErrCh := vpphelper.StartAndDialContext(connectCtx)
+``` 
+The following two functional options can also be passed as parameters to the `StartAndDialContext` function:  
+1. `WithRootDir` : sets the root directory for all `.conf` files
+2. `WithVppConfig` : sets `vpp.conf` file template. All the `%[1]s` in the template will be replaced by the `rootDir`.
+```go
+conn, vppErrCh := vpphelper.StartAndDialContext(connectCtx, vpphelper.WithRootDir("/usr"), vpphelper.WithVppConfig(newVppConfContents))
```

Comment:
Could we change the example rootdir from /usr to something more appropriate like "/tmp/vpp2" ?

Reply:
Made this change? Any other changes needed?

Should I merge all the commits to one commit and then force push?

Reply:
> Should I merge all the commits to one commit and then force push?